docs/introduction.md

```diff
@@ -5,402 +5,103 @@ This document is aimed at introducing developers to the Flow Data Provisioning S
+* Its root checkpoint
+* The protocol state database from one of its consensus nodes
+* The write-ahead log from one of its execution nodes
 
-Layer-2 solutions include networks like Bitcoin's Lightning and Ethereum's Plasma — these networks work off the main chain.
+You can then run the `flow-dps-indexer` binary, giving it access to those elements like such:
 
-Sharding is a technique where a network is broken up into several interconnected networks.
-Sharding significantly increases the complexity of the programming model by breaking ACID guarantees (Atomicity, Consistency, Isolation and Durability), increasing the cost and time for application development.
+```console
+$ ./flow-dps-indexer -a -l debug -d /var/flow/data/protocol -t /var/flow/data/execution -c /var/flow/bootstrap/root.checkpoint -i /var/flow/data/index
+```
 
-Flow was designed to provide a blockchain that can scale while preserving composability in a single blockchain state.
-This is achieved by a novel approach where work traditionally assigned to full nodes is split and assigned to specific roles, allowing pipelining.
-We recognize the following roles in the Flow architecture.
+This can take a very long time to process, and require large amounts of available storage space for the new index to be created.
+Once the indexing process is over, you can use the created DPS index to serve the DPS API, by running `flow-dps-server` and giving it access to the index.
 
-1. Collection role — in charge of transaction collection from the user agents
-2. Execution role — in charge of executing the transactions
-3. Consensus role — maintain the chain of blocks and responsible for the chain extension by appending new blocks. These nodes also rule on reported misbehaviors of other nodes
-4. Verification role — done by a more extensive set of verification nodes, they confirm that the execution results are correct
-5. Access role — also called Observer role — includes nodes that relay data to protocol-external entities that are not participating in the protocol
+```console
+$ ./flow-dps-server -i /var/flow/data/index -a localhost:5005
+```
 
-Using those [node roles](#flow-node-roles), each type of node can be optimized according to the tasks it performs.
-For instance, execution nodes are compute-optimized nodes, leveraging large-scale data centers, while collection nodes are highly bandwidth-optimized.
-Goal for the Flow blockchain is to allow a high degree of participation for the consensus and verification nodes by having only moderate hardware requirements.
-These nodes would only require a high-end consumer internet connection.
+You should now have the DPS API available at `localhost:5005`.
+It can be used in conjunction with the DPS Rosetta Server and the DPS Access API, which both need the address to your DPS API in order to function.
```

Comment:
Let's stick with `Flow Rosetta API` and `Flow Access API` for now. We most likely will end up as the default reference, so we can just claim the name without adding DPS.